packetbeat/Makefile

```diff
@@ -14,9 +14,13 @@ with_pfring:
 before-build:
 	sed -i.bk 's/device: any/device: en0/' $(PREFIX)/packetbeat-darwin.yml
 	rm $(PREFIX)/packetbeat-darwin.yml.bk
+	sed -i.bk 's/device: any/device: en0/' $(PREFIX)/packetbeat-darwin.full.yml
+	rm $(PREFIX)/packetbeat-darwin.full.yml.bk
```

Comment:
Should we do the same for filebeat and the windows version for the paths? Currently we have:

```
  paths:
    - /var/log/*.log
    #- c:\programdata\elasticsearch\logs\*
```